packages/react-topology/src/components/popper/Portal.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
+import { useIsomorphicLayoutEffect } from '../../../../react-core/src/helpers';
```

Comment:
Change to `from '@patternfly/react-core/dist/esm/helpers';`

Reply:
![image](https://user-images.githubusercontent.com/52387464/123850326-82481200-d8df-11eb-8d40-ccef7a75de4f.png)

Reply:
Ah my mistake. Try just `from '@patternfly/react-core';`.